File: TUnit.TestProject/TestDataSources.cs
namespace TUnit.TestProject;

public class TestDataSources
{
    public static int One() => 1;
    public static int Two() => 2;

    public static int[] OneEnumerable() => [1, 1, 1, 1, 1, 1, 1, 1, 1, 1];
    public static int[] OneFailingEnumerable() => [1, 2, 3, 4, 5, 6, 7, 8, 9];

}

File: TUnit.TestProject.FSharp/TestDataSources.fs
namespace TUnit.TestProject.FSharp

type TestDataSources() =
    static member One() = 1
    static member Two() = 2

    static member OneEnumerable() = [| 1; 1; 1; 1; 1; 1; 1; 1; 1; 1 |]
    static member OneFailingEnumerable() = [| 1; 2; 3; 4; 5; 6; 7; 8; 9 |]

